unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs

```diff
@@ -1,29 +1,24 @@
-using System;
+using UnityEngine;
```

Comment:
Please remove the formating/cleanup rearrangment of the code in this class.

It's hard to read the changes.